practice/batch/services/services_actions.rb

```diff
@@ -0,0 +1,18 @@
+module SERVICES
+  STOPPED = 'STOPPED'.freeze
+  RUNNING = 'RUNNING'.freeze
+
+
+  def self.service_status(service_name, status)
```

Comment:
I see this method in a bit different way. service_status should accept only one argument 'service_name' and return its state.